newEvaluation.html

```diff
@@ -0,0 +1,394 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>New Evaluation</title>
+	<link rel="stylesheet" type="text/css" href="newEvaluation.css">
+	<link rel="stylesheet" type="text/css" href="common.css">
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+	<!-- Nav Bar -->
+	<nav>
+		<img class = "logoPicture" src = "./assets/logo.jpg">
+		<a href = "evaluations.html">Evaluations</a>
+		<a href = "newEvaluations.html">New Evaluation</a>
+		<a id = "logout" href = "logout.html">Logout</a>
+	</nav>
+	
+	<!-- Section1 -->
+	<div class = "fieldset-container">
+	<form>
+	<section>
+		<input type = "name" name = "candidate" placeholder = "Candidate">
+		<input type = "name" name = "interviewer" placeholder = "Interviewer">
+		<input type = "date" name = "date" placeholder = date()>
+	</section>
+	
+	<!-- Section2 -->
+	<section>
+		<h5>Technical Level</h5>
```

Comment:
Initially I thought it is the same size as the pdf-s heading. Is it recommended to use h2 for sections headings?